examples/planning/closed_loop_test.ipynb

```diff
@@ -0,0 +1,347 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Closed-Loop Evaluation\n",
+    "In this notebook you are going to evaluate a CNN-based policy to control the SDV with a protocol named *closed-loop* evaluation.\n",
+    "\n",
+    "**Note: this notebook assumes you've already run the [training notebook](./train.ipynb) and stored your model successfully.**\n",
+    "\n",
+    "## What is a closed-loop evaluation?\n",
+    "In closed-loop evaluation the model is in **full control of the SDV**. At each time step, we predict the future trajectory and then move the AV to the first of the model's predictions. \n",
+    "\n",
+    "We refer to this process with the terms **forward-simulate** or **unroll**.\n",
+    "\n",
+    "![closed-loop](../../images/planning/closed-loop.svg)\n",
+    "\n",
+    "\n",
+    "## What can we use closed-loop evaluation for?\n",
+    "Closed-loop is crucial to assess the real performance of a model. **Ideally, we would test the model on the road in the real world**. However, this is clearly very expensive and scales poorly. Forward-simulation is an attempt to evaluate the system in a setting which is as close as possible to a real road test on the same route.\n",
+    "\n",
+    "Differently from the open-loop, the model is now in full control of the SDV and predictions are used to compute the future location of the SDV.\n",
+    "\n",
+    "## Is closed-loop evaluation enough?\n",
+    "Closed-loop evaluation is an important step forward towards evaluating how our policy would perform on the road. However, it still have some limitations.\n",
+    "\n",
+    "The critical one is the **non-reactivity of other traffic participants**. In fact, while the SDV is now controlled by our policy, other actors are still being replayed from the original log. In this setting, a chasing car will not slow down if our policy choses a different speed profile for the SDV, resulting in a rear collision that wouldn't obviously occur in the real world.\n",
+    "\n",
+    "For this reason, closed-loop evaluation is only effective for the first few seconds of forward simulation."
```

Comment:
Leaving the reader kind of hanging here, maybe 'resolve' the situation? E.g. say: but this is not bad because we have ...

Reply:
added a sentence